features/update/remove_price.feature

```diff
@@ -16,7 +16,7 @@ Feature: Remove price fields
       | BOX4 | 5 EUR, 5 USD  | 5 EUR, 5 USD  |
     Then I should get the following products after apply the following updater to it:
       | product | actions                                                                                                                                                                                                                                                | result                                                                                                                                                                                                                          |
-      | BOX1    | [{"type": "remove_data", "field": "price", "data": [{"amount": "", "currency": "EUR"}], "locale": "fr_FR", "scope": null}]                                                                                                                             | {"values": {"price": [{"locale": "fr_FR", "scope": null, "data": [{"amount": "5", "currency": "USD"}]}, {"locale": "en_US", "scope": null, "data": [{"amount": "5", "currency": "EUR"}, {"amount": "5", "currency": "USD"}]}]}} |
```

Comment:
The before and after results are the same. I just inverted the order in wich the prices in the price collection appears.